### PR TITLE
markup update as per user guide review comments by Robin

### PR DESCRIPTION
**Pull request by itsprashantlab**

### CODE REVIEW

File: docs/Zendesk-batchsource.md [hunk 1]
```diff
@@ -48,19 +47,19 @@ the specified date. The date must be provided in the date format:
 |                                  | YYYY-MM-DDThh:mm:ss-hh:mm | 1999-01-01T23:01:01-08:00 |
 |                                  | YYYY-MM-DDThh:mm:ssZ      | 1999-01-01T23:01:01Z      |
 
-Specifying this along with `Start Date` allows reading data modified within a specific time window.
+If you enter an End Date and Start Date, the data modified within a specific time window.
```
Comment (robinrielley) [inline docs/Zendesk-batchsource.md]: I think we are missing "is: after "data".  "...the data is modified within a specific time window."

File: docs/Zendesk-batchsource.md [hunk 2]
```diff
@@ -48,19 +47,19 @@ the specified date. The date must be provided in the date format:
...
 If no value is provided, no upper bound is applied.
 
-**Satisfaction Ratings Score:** Filter Satisfaction Ratings object to include only records which have Zendesk score
-equal to the specified score.
+**Satisfaction Ratings Score:** Filter Satisfaction Ratings object to include only records that have a Zendesk score
+equal to the specified score. Only applicable for the Satisfaction Rating object
```
Comment (robinrielley) [inline docs/Zendesk-batchsource.md]: Can you add a period after "object" at the end of the sentence?

File: docs/Zendesk-batchsource.md [hunk 3]
```diff
@@ -48,19 +47,19 @@ the specified date. The date must be provided in the date format:
...
 
 ### Advanced
 
-**Max Retry Count:** Maximum number of retry attempts.
+**Max Retry Count:** Maximum number of retry attempts.Default is 20.
```
Comment (robinrielley) [inline docs/Zendesk-batchsource.md]: Can you add a space after the period?

File: docs/Zendesk-batchsource.md [hunk 4]
```diff
@@ -48,19 +47,19 @@ the specified date. The date must be provided in the date format:
...
 
-**Connect Timeout:** Maximum time in seconds connection initialization can take.
+**Connect Timeout:** Maximum time in seconds connection initialization. Default is 300.
```
Comment (robinrielley) [inline docs/Zendesk-batchsource.md]: I think you are missing the word "for".  Should be: Maximum time in seconds for connection initialization.

File: docs/Zendesk-batchsource.md [hunk 5]
```diff
@@ -48,19 +47,19 @@ the specified date. The date must be provided in the date format:
...
 
-**Read Timeout:** Maximum time in seconds fetching data from the server can take.
+**Read Timeout:** Maximum time in seconds fetching data from the server. Default is 300.
```
Comment (robinrielley) [inline docs/Zendesk-batchsource.md]: This is the wording in our Google doc: Maximum time in seconds to fetch data from the server.

File: docs/Zendesk-batchsource.md [hunk 6]
```diff
@@ -39,7 +38,7 @@ or equal to the specified date. The date must be provided in the date format:
 
 Start Date is required for batch objects like: Ticket Comments, Organizations, Ticket Metric Events, Tickets, Users.
 
-**End Date:** Filter data to include only records which have Zendesk modified date is less than
+**End Date:** Filter data to include only records which have Zendesk modified date that is less than
```
Comment (robinrielley) [inline docs/Zendesk-batchsource.md]: Here's what's in the doc: Filter data to include only records that have a Zendesk modified date that is less than the specified date.

File: docs/Zendesk-batchsource.md [hunk 7]
```diff
@@ -16,19 +16,18 @@ Configuration
...
-Check out [Zendesk documentation](https://support.zendesk.com/hc/en-us/articles/226022787-Generating-a-new-API-token-)
-for API Token generation.
+For API Token generation, see the [Zendesk documentation](https://support.zendesk.com/hc/en-us/articles/226022787-Generating-a-new-API-token-).
 
 **Subdomains:** List of Zendesk Subdomains to read object from.
 
 **Object to Pull:** Objects to pull from Zendesk API.
 
-**Start Date:** Filter data to include only records which have Zendesk modified date is greater than
+**Start Date:** Filter data to include only records which have Zendesk modified date that is greater than
```
Comment (robinrielley) [inline docs/Zendesk-batchsource.md]: Here's what's in the doc: Filter data to include only records that have a Zendesk modified date that is greater than or equal to the specified date.

File: docs/Zendesk-batchsource.md (hunk 4 above)
Comment (robinrielley) [inline docs/Zendesk-batchsource.md]: Missing "for". Here's the sentence in the docs: Maximum time in seconds for connection initialization.

File: docs/Zendesk-batchsource.md (hunk 1 above)
Comment (prashantgoogle) [reply docs/Zendesk-batchsource.md]: added

File: docs/Zendesk-batchsource.md (hunk 2 above)
Comment (prashantgoogle) [reply docs/Zendesk-batchsource.md]: updated

File: docs/Zendesk-batchsource.md (hunk 3 above)
Comment (prashantgoogle) [reply docs/Zendesk-batchsource.md]: updated

File: docs/Zendesk-batchsource.md (hunk 4 above)
Comment (prashantgoogle) [reply docs/Zendesk-batchsource.md]: updated

File: docs/Zendesk-batchsource.md (hunk 5 above)
Comment (prashantgoogle) [reply docs/Zendesk-batchsource.md]: updated

File: docs/Zendesk-batchsource.md (hunk 6 above)
Comment (prashantgoogle) [reply docs/Zendesk-batchsource.md]: updated

File: docs/Zendesk-batchsource.md (hunk 7 above)
Comment (prashantgoogle) [reply docs/Zendesk-batchsource.md]: updated

File: docs/Zendesk-batchsource.md (hunk 4 above)
Comment (prashantgoogle) [reply docs/Zendesk-batchsource.md]: updated

File: docs/Zendesk-batchsource.md [hunk 8]
```diff
@@ -16,19 +16,18 @@ Configuration
...
 **Admin Email:** Zendesk admin email.
 
 **API Token:** Zendesk API token. Can be obtained from the Zendesk Support Admin interface.
-Check out [Zendesk documentation](https://support.zendesk.com/hc/en-us/articles/226022787-Generating-a-new-API-token-)
-for API Token generation.
+For API Token generation, see the [Zendesk documentation](https://support.zendesk.com/hc/en-us/articles/226022787-Generating-a-new-API-token-).
 
 **Subdomains:** List of Zendesk Subdomains to read object from.
 
 **Object to Pull:** Objects to pull from Zendesk API.
```
Comment (robinrielley) [inline docs/Zendesk-batchsource.md]: add "Default is Groups."

File: docs/Zendesk-batchsource.md [hunk 9]
```diff
@@ -39,7 +38,7 @@ or equal to the specified date. The date must be provided in the date format:
 
 Start Date is required for batch objects like: Ticket Comments, Organizations, Ticket Metric Events, Tickets, Users.
 
-**End Date:** Filter data to include only records which have Zendesk modified date is less than
+**End Date:** Filter data to include only records which have a Zendesk modified date that is less than
```
Comment (robinrielley) [inline docs/Zendesk-batchsource.md]: can you change "which" to "that". Filter data to include only records that have a Zendesk modified date that is ...."

File: docs/Zendesk-batchsource.md [hunk 10]
```diff
@@ -16,19 +16,18 @@ Configuration
...
-Check out [Zendesk documentation](https://support.zendesk.com/hc/en-us/articles/226022787-Generating-a-new-API-token-)
-for API Token generation.
+For API Token generation, see the [Zendesk documentation](https://support.zendesk.com/hc/en-us/articles/226022787-Generating-a-new-API-token-).
 
 **Subdomains:** List of Zendesk Subdomains to read object from.
 
 **Object to Pull:** Objects to pull from Zendesk API.
 
-**Start Date:** Filter data to include only records which have Zendesk modified date is greater than
+**Start Date:** Filter data to include only records which have a Zendesk modified date that is greater than
```
Comment (robinrielley) [inline docs/Zendesk-batchsource.md]: can you change "which" to "that". Filter data to include only records that have a Zendesk modified date that is

File: docs/Zendesk-batchsource.md [hunk 11]
```diff
@@ -48,19 +47,19 @@ the specified date. The date must be provided in the date format:
...
+**Max Retry Count:** Maximum number of retry attempts. Default is 20.
 
-**Connect Timeout:** Maximum time in seconds connection initialization can take.
+**Connect Timeout:** Maximum time in seconds for connection initialization. Default is 300.
 
-**Read Timeout:** Maximum time in seconds fetching data from the server can take.
+**Read Timeout:** Maximum time in seconds to fetch data from the server. Default is 300.
 
 Data Type Mappings from Zendesk to CDAP
 ----------
```
Comment (robinrielley) [inline docs/Zendesk-batchsource.md:68]: The markdown for Limitations is different than the Limitations content in the Google doc. Which is correct?
See https://docs.google.com/document/d/1cNQqpMcEQXfN2aBDpVYCiBGaTyfh8W4WAkQXoOVgvdY/edit#

**Review by robinrielley — CHANGES_REQUESTED** [review]
I've added line edits. The Limitations section in the markdown is different than the Google doc. Which one is correct?  https://docs.google.com/document/d/1cNQqpMcEQXfN2aBDpVYCiBGaTyfh8W4WAkQXoOVgvdY/edit#

File: docs/Zendesk-batchsource.md (hunk 9 above)
Comment (prashantgoogle) [reply docs/Zendesk-batchsource.md]: updated

File: docs/Zendesk-batchsource.md (hunk 10 above)
Comment (prashantgoogle) [reply docs/Zendesk-batchsource.md]: updated

File: docs/Zendesk-batchsource.md (hunk 11 above)
Comment (prashantgoogle) [reply docs/Zendesk-batchsource.md:68]: updated

File: docs/Zendesk-batchsource.md [hunk 12]
```diff
@@ -77,8 +79,17 @@ corresponding CDAP types.
...
+
+Zendesk supports three types of pagination: [offset](https://developer.zendesk.com/documentation/developer-tools/pagination/paginating-through-lists-using-offset-pagination/), [time-based](https://developer.zendesk.com/documentation/developer-tools/pagination/paginating-through-lists-using-cursor-pagination/) and [cursor](https://developer.zendesk.com/documentation/developer-tools/pagination/paginating-through-lists-using-cursor-pagination/) pagination. 
+The following issues occur when the Zendesk API uses these pagination.
+- Offset and Time-Based Pagination: Data Duplicacy
+- Cursor Pagination: Data Loss
+
+### Data Duplication issues with Zendesk APIs
+
 This plugin can return duplicate records for time-based exports as mentioned in Zendesk docs.
```
Comment (robinrielley) [inline docs/Zendesk-batchsource.md]: We need to update this section to match the changes in Limitations section: offset also applies: https://docs.google.com/document/d/1cNQqpMcEQXfN2aBDpVYCiBGaTyfh8W4WAkQXoOVgvdY/edit#heading=h.gjdgxs

File: docs/ZendeskMultiObjects-batchsource.md [hunk 13]
```diff
@@ -56,19 +57,25 @@ the specified date. The date must be provided in the date format:
 Specifying this along with `Start Date` allows reading data modified within a specific time window. 
 If no value is provided, no upper bound is applied.
 
-**Satisfaction Ratings Score:** Filter Satisfaction Ratings object to include only records which have Zendesk score
-equal to the specified score.
+**Satisfaction Ratings Score:** Filter Satisfaction Ratings object to include only records which have Zendesk score equal to the specified score. 
+Only Applicable for Satisfaction Ratings object. 
+The satisfaction ratings score cannot be a numerical range.
```
Comment (robinrielley) [inline docs/ZendeskMultiObjects-batchsource.md]: This is impossible to do in the UI. I don't think we should have this in the doc. There's actually no way to do this.

File: docs/ZendeskMultiObjects-batchsource.md [hunk 14]
```diff
@@ -24,14 +24,15 @@ Configuration
...
 **API Token:** Zendesk API token. Can be obtained from the Zendesk Support Admin interface.
-Check out [Zendesk documentation](https://support.zendesk.com/hc/en-us/articles/226022787-Generating-a-new-API-token-)
-for API Token generation.
+For information about generating the Zendesk API token, see [Zendesk API Token](https://support.zendesk.com/hc/en-us/articles/226022787-Generating-a-new-API-token-).
 
-**Subdomains:** List of Zendesk Subdomains to read object from.
+**Subdomains:** List of Zendesk [Subdomains](https://support.zendesk.com/hc/en-us/articles/4409381383578-Where-can-I-find-my-Zendesk-subdomain-) to read object from.
 
-**Object to Pull:** Objects to pull from Zendesk API.
+**Object to Pull:** Objects to pull from Zendesk API.If not specified, data is pulled from all objects. Default is blank.
```
Comment (robinrielley) [inline docs/ZendeskMultiObjects-batchsource.md]: Missing a space after the period "...API. If"

File: docs/ZendeskMultiObjects-batchsource.md [hunk 15]
```diff
@@ -56,19 +57,25 @@ the specified date. The date must be provided in the date format:
 Specifying this along with `Start Date` allows reading data modified within a specific time window. 
 If no value is provided, no upper bound is applied.
 
-**Satisfaction Ratings Score:** Filter Satisfaction Ratings object to include only records which have Zendesk score
-equal to the specified score.
+**Satisfaction Ratings Score:** Filter Satisfaction Ratings object to include only records which have Zendesk score equal to the specified score. 
+Only Applicable for Satisfaction Ratings object. 
+The satisfaction ratings score cannot be a numerical range. 
+Instead, select one of the available scores in the dropdown menu when you configure the plugin.
```
Comment (robinrielley) [inline docs/ZendeskMultiObjects-batchsource.md]: I'm deleting this for CDAP.

File: docs/ZendeskMultiObjects-batchsource.md [hunk 16]
```diff
@@ -56,19 +57,25 @@ the specified date. The date must be provided in the date format:
 Specifying this along with `Start Date` allows reading data modified within a specific time window. 
 If no value is provided, no upper bound is applied.
 
-**Satisfaction Ratings Score:** Filter Satisfaction Ratings object to include only records which have Zendesk score
-equal to the specified score.
+**Satisfaction Ratings Score:** Filter Satisfaction Ratings object to include only records which have Zendesk score equal to the specified score. 
+Only Applicable for Satisfaction Ratings object. 
+The satisfaction ratings score cannot be a numerical range. 
+Instead, select one of the available scores in the dropdown menu when you configure the plugin. 
+Example value: Good With Comment
```
Comment (robinrielley) [inline docs/ZendeskMultiObjects-batchsource.md]: I'm deleting this for CDAP.

File: docs/ZendeskMultiObjects-batchsource.md [hunk 17]
```diff
@@ -56,19 +57,25 @@ the specified date. The date must be provided in the date format:
...
 
 **Table Name Field:** The name of the field that holds the table name. Must not be the name of any table column that
 will be read. Defaults to `tablename`.
 
 ### Advanced
 
-**Max Retry Count:** Maximum number of retry attempts.
+**Max Retry Count:** Maximum number of retry attempts.Maximum number of retry attempts. Default is 20.
```
Comment (robinrielley) [inline docs/ZendeskMultiObjects-batchsource.md]: Missing space after period "...attempt. Maximum..."

File: docs/ZendeskMultiObjects-batchsource.md [hunk 18]
```diff
@@ -56,19 +57,25 @@ the specified date. The date must be provided in the date format:
...
+
+
+**Connect Timeout:** Maximum time in seconds that connection initialization can take.Default is 300.
```
Comment (robinrielley) [inline docs/ZendeskMultiObjects-batchsource.md]: Missing space after period "...take. Default..."

File: docs/ZendeskMultiObjects-batchsource.md [hunk 19]
```diff
@@ -56,19 +57,25 @@ the specified date. The date must be provided in the date format:
...
 
-**Connect Timeout:** Maximum time in seconds connection initialization can take.
 
-**Read Timeout:** Maximum time in seconds fetching data from the server can take.
+**Read Timeout:** Maximum time in seconds that fetching data from the server can take.Default is 300.
```
Comment (robinrielley) [inline docs/ZendeskMultiObjects-batchsource.md]: Missing space after period "..take. Default"

File: docs/ZendeskMultiObjects-batchsource.md [hunk 20]
```diff
@@ -85,16 +92,24 @@ corresponding CDAP types.
...
+- Cursor Pagination: Data Loss
+
+### Data Duplication issues with Zendesk APIs
+
+The Zendesk Multi Batch Source plugin might return duplicate records for offset and time-based exports as mentioned in Zendesk docs.
 [Zendesk documentation](https://developer.zendesk.com/documentation/ticketing/managing-tickets/using-the-incremental-export-api/#excluding-duplicate-items)
 
-Objects which support time-based exports:
+Objects that support offset and time-based exports:
 **Users**,
```
Comment (robinrielley) [inline docs/ZendeskMultiObjects-batchsource.md]: You can unbold this list.

File: docs/ZendeskMultiObjects-batchsource.md [hunk 21]
```diff
@@ -24,14 +24,15 @@ Configuration
 **Admin Email:** Zendesk admin email.
```
Comment (robinrielley) [inline docs/ZendeskMultiObjects-batchsource.md:27]: The description for this plugin doesn't match the Google doc. The Zendesk Batch Source markdown has the updated overview. https://docs.google.com/document/d/1T8eCPI9rgZm7tL9OfhWb2EtUGfdhMYY7aryfos7deKM/edit?resourcekey=0-JCNeuQYONXE8cB9WS3iEJw#

Also, the name of the plugin should be Zendesk Multi Objects Batch Source. The current name reads without a space Zendesk MultiObjects Batch Source

File: docs/ZendeskMultiObjects-batchsource.md [hunk 22]
```diff
@@ -24,14 +24,15 @@ Configuration
...
 
-**Object to Pull:** Objects to pull from Zendesk API.
+**Object to Pull:** Objects to pull from Zendesk API.If not specified, data is pulled from all objects. Default is blank.
+
+
+**Object to Skip:** Objects to skip from Zendesk API. Default is blank.
 
-**Object to Skip:** Objects to Skip from Zendesk API.
 
 **Start Date:** Filter data to include only records which have Zendesk modified date is greater than
```
Comment (robinrielley) [inline docs/ZendeskMultiObjects-batchsource.md]: can you change which to "that"

Comment (robinrielley) [reply docs/ZendeskMultiObjects-batchsource.md]: Same for End Date

**Review by robinrielley — CHANGES_REQUESTED** [review]
I've added comments. Can I have one final review after these are fixed?

File: docs/Zendesk-batchsource.md [hunk 23]
```diff
@@ -77,16 +79,24 @@ corresponding CDAP types.
 | Array                  | Array     |
 | Record                 | Record    |
 
-Data Duplication issues with Zendesk APIs
+
+Limitations
 ----------
-This plugin can return duplicate records for time-based exports as mentioned in Zendesk docs.
+
+Zendesk supports three types of pagination: [offset](https://developer.zendesk.com/documentation/developer-tools/pagination/paginating-through-lists-using-offset-pagination/), [time-based](https://developer.zendesk.com/documentation/ticketing/managing-tickets/using-the-incremental-export-api/#time-based-incremental-exports)  pagination.
```
Comment (robinrielley) [inline docs/Zendesk-batchsource.md]: looks like we removed Cursor Pagination: Results in Data Loss
but didn't change Zendesk supports three types of... to "two types of"

Comment (robinrielley) [reply docs/Zendesk-batchsource.md]: Can you delete the comma before time-based and just say "offset and time-based"

File: docs/ZendeskMultiObjects-batchsource.md [hunk 24]
```diff
@@ -85,16 +89,23 @@ corresponding CDAP types.
 | Array                  | Array     |
 | Record                 | Record    |
 
-Data Duplication issues with Zendesk APIs
+Limitations
 ----------
-This plugin can return duplicate records for time-based exports as mentioned in Zendesk docs.
+
+Zendesk supports three types of pagination: [offset](https://developer.zendesk.com/documentation/developer-tools/pagination/paginating-through-lists-using-offset-pagination/), [time-based](https://developer.zendesk.com/documentation/ticketing/managing-tickets/using-the-incremental-export-api/#time-based-incremental-exports)  pagination.
```
Comment (robinrielley) [inline docs/ZendeskMultiObjects-batchsource.md]: change to "two types". and "and" rather than a comma between the two types

**Review by robinrielley — CHANGES_REQUESTED** [review]
added new comments. old ones still outstanding

File: docs/Zendesk-batchsource.md (hunk 12 above)
Comment (vikasrathee-cs) [reply docs/Zendesk-batchsource.md]: done

File: docs/ZendeskMultiObjects-batchsource.md (hunk 13 above)
Comment (vikasrathee-cs) [reply docs/ZendeskMultiObjects-batchsource.md]: removed

File: docs/ZendeskMultiObjects-batchsource.md (hunk 14 above)
Comment (vikasrathee-cs) [reply docs/ZendeskMultiObjects-batchsource.md]: done

File: docs/ZendeskMultiObjects-batchsource.md (hunk 15 above)
Comment (vikasrathee-cs) [reply docs/ZendeskMultiObjects-batchsource.md]: done

File: docs/ZendeskMultiObjects-batchsource.md (hunk 16 above)
Comment (vikasrathee-cs) [reply docs/ZendeskMultiObjects-batchsource.md]: done

File: docs/ZendeskMultiObjects-batchsource.md (hunk 17 above)
Comment (vikasrathee-cs) [reply docs/ZendeskMultiObjects-batchsource.md]: done

File: docs/ZendeskMultiObjects-batchsource.md (hunk 18 above)
Comment (vikasrathee-cs) [reply docs/ZendeskMultiObjects-batchsource.md]: done

File: docs/ZendeskMultiObjects-batchsource.md (hunk 19 above)
Comment (vikasrathee-cs) [reply docs/ZendeskMultiObjects-batchsource.md]: done

File: docs/ZendeskMultiObjects-batchsource.md (hunk 20 above)
Comment (vikasrathee-cs) [reply docs/ZendeskMultiObjects-batchsource.md]: done

File: docs/ZendeskMultiObjects-batchsource.md (hunk 21 above)
Comment (vikasrathee-cs) [reply docs/ZendeskMultiObjects-batchsource.md:27]: done

File: docs/ZendeskMultiObjects-batchsource.md (hunk 22 above)
Comment (vikasrathee-cs) [reply docs/ZendeskMultiObjects-batchsource.md]: done

File: docs/Zendesk-batchsource.md (hunk 23 above)
Comment (vikasrathee-cs) [reply docs/Zendesk-batchsource.md]: done

File: docs/ZendeskMultiObjects-batchsource.md (hunk 24 above)
Comment (vikasrathee-cs) [reply docs/ZendeskMultiObjects-batchsource.md]: done

**Comment by rmstar** [comment]
Please take a look at e2e test failures

File: docs/Zendesk-batchsource.md [hunk 25]
```diff
@@ -3,7 +3,10 @@
 
 Description
 -----------
-This source reads objects from Zendesk.
+This source reads objects from Zendesk.It extracts reportable data from the Zendesk objects.
```
Comment (robinrielley) [inline docs/Zendesk-batchsource.md]: missing a space after the period.

File: docs/Zendesk-batchsource.md [hunk 26]
```diff
@@ -3,7 +3,10 @@
 
 Description
 -----------
-This source reads objects from Zendesk.
+This source reads objects from Zendesk.It extracts reportable data from the Zendesk objects. 
+The Zendesk Batch Source plugin enables bulk data extraction from Zendesk. 
+You can configure and execute bulk data transfers from Zendesk without any coding
```
Comment (robinrielley) [inline docs/Zendesk-batchsource.md]: missing period at the end of the sentence

File: docs/Zendesk-batchsource.md [hunk 27]
```diff
@@ -77,16 +79,44 @@ corresponding CDAP types.
...
+Limitations
 ----------
-This plugin can return duplicate records for time-based exports as mentioned in Zendesk docs.
+
+Zendesk plugin supports two types of pagination: [offset](https://developer.zendesk.com/documentation/developer-tools/pagination/paginating-through-lists-using-offset-pagination/) and [time-based](https://developer.zendesk.com/documentation/ticketing/managing-tickets/using-the-incremental-export-api/#time-based-incremental-exports)  pagination.
+Offset and Time-Based Pagination might result in data duplication.
+
+### Data Duplication issues with Zendesk APIs
+
+This plugin can return duplicate records for offset and time-based exports as mentioned in Zendesk docs.
```
Comment (robinrielley) [inline docs/Zendesk-batchsource.md]: change "can" to "might"

File: docs/ZendeskMultiObjects-batchsource.md [hunk 28]
```diff
@@ -1,9 +1,12 @@
-# Zendesk MultiObjects Batch Source
+# Zendesk Multi Objects Batch Source
 
 
 Description
 -----------
-This source reads multiple objects from Zendesk.
+This source reads multiple objects from Zendesk.It extracts reportable data from the multiple Zendesk objects.
```
Comment (robinrielley) [inline docs/ZendeskMultiObjects-batchsource.md]: add space after period

File: docs/ZendeskMultiObjects-batchsource.md [hunk 29]
```diff
@@ -1,9 +1,12 @@
-# Zendesk MultiObjects Batch Source
+# Zendesk Multi Objects Batch Source
 
 
 Description
 -----------
-This source reads multiple objects from Zendesk.
+This source reads multiple objects from Zendesk.It extracts reportable data from the multiple Zendesk objects. 
+The Zendesk Multi Object Batch Source plugin enables bulk data extraction from Zendesk. 
+You can configure and execute bulk data transfers from Zendesk without any coding
```
Comment (robinrielley) [inline docs/ZendeskMultiObjects-batchsource.md]: add period at the end of the sentence

File: docs/ZendeskMultiObjects-batchsource.md [hunk 30]
```diff
@@ -44,7 +46,7 @@ or equal to the specified date. The date must be provided in the date format:
 
 Start Date is required for batch objects like: Ticket Comments, Organizations, Ticket Metric Events, Tickets, Users.
 
-**End Date:** Filter data to include only records which have Zendesk modified date is less than 
+**End Date:** Filter data to include only records that have Zendesk modified date is less than
```
Comment (robinrielley) [inline docs/ZendeskMultiObjects-batchsource.md]: Missing some words:
"Filter data to include only records that have a Zendesk modified date that is less than...."
See: https://docs.google.com/document/d/1T8eCPI9rgZm7tL9OfhWb2EtUGfdhMYY7aryfos7deKM/edit?resourcekey=0-JCNeuQYONXE8cB9WS3iEJw#

File: docs/ZendeskMultiObjects-batchsource.md [hunk 31]
```diff
@@ -53,22 +55,25 @@ the specified date. The date must be provided in the date format:
 |                                  | YYYY-MM-DDThh:mm:ss-hh:mm | 1999-01-01T23:01:01-08:00 |
 |                                  | YYYY-MM-DDThh:mm:ssZ      | 1999-01-01T23:01:01Z      |
 
-Specifying this along with `Start Date` allows reading data modified within a specific time window. 
+If you enter an End Date and Start Date, the data is modified within a specific time window. 
 If no value is provided, no upper bound is applied.
 
-**Satisfaction Ratings Score:** Filter Satisfaction Ratings object to include only records which have Zendesk score
-equal to the specified score.
+**Satisfaction Ratings Score:** Filter Satisfaction Ratings object to include only records that have Zendesk score equal to the specified score.
```
Comment (robinrielley) [inline docs/ZendeskMultiObjects-batchsource.md]: add "a".  that have a Zendesk score"

File: docs/ZendeskMultiObjects-batchsource.md [hunk 32]
```diff
@@ -53,22 +55,25 @@ the specified date. The date must be provided in the date format:
...
+**Satisfaction Ratings Score:** Filter Satisfaction Ratings object to include only records that have Zendesk score equal to the specified score. 
+Only Applicable for Satisfaction Ratings object.
 
 **Table Name Field:** The name of the field that holds the table name. Must not be the name of any table column that
 will be read. Defaults to `tablename`.
 
 ### Advanced
 
-**Max Retry Count:** Maximum number of retry attempts.
+**Max Retry Count:** Maximum number of retry attempts. Maximum number of retry attempts. Default is 20.
```
Comment (robinrielley) [inline docs/ZendeskMultiObjects-batchsource.md]: you have Maximum number of retry attems 2x. Delete one.

File: docs/ZendeskMultiObjects-batchsource.md [hunk 33]
```diff
@@ -85,16 +90,99 @@ corresponding CDAP types.
...
+Limitations
 ----------
-This plugin can return duplicate records for time-based exports as mentioned in Zendesk docs.
+
+Zendesk plugin supports two types of pagination: [offset](https://developer.zendesk.com/documentation/developer-tools/pagination/paginating-through-lists-using-offset-pagination/) and [time-based](https://developer.zendesk.com/documentation/ticketing/managing-tickets/using-the-incremental-export-api/#time-based-incremental-exports)  pagination.
+Offset and Time-Based Pagination might result in data duplication.
+
+### Data Duplication issues with Zendesk APIs
+
+The Zendesk Multi Batch Source plugin might return duplicate records for offset and time-based exports as mentioned in Zendesk docs.
```
Comment (robinrielley) [inline docs/ZendeskMultiObjects-batchsource.md:101]: delete extra space after "might"

File: docs/Zendesk-batchsource.md (hunk 25 above)
Comment (vikasrathee-cs) [reply docs/Zendesk-batchsource.md]: done

File: docs/Zendesk-batchsource.md (hunk 26 above)
Comment (vikasrathee-cs) [reply docs/Zendesk-batchsource.md]: done

File: docs/Zendesk-batchsource.md (hunk 27 above)
Comment (vikasrathee-cs) [reply docs/Zendesk-batchsource.md]: done

File: docs/ZendeskMultiObjects-batchsource.md (hunk 28 above)
Comment (vikasrathee-cs) [reply docs/ZendeskMultiObjects-batchsource.md]: done

File: docs/ZendeskMultiObjects-batchsource.md (hunk 29 above)
Comment (vikasrathee-cs) [reply docs/ZendeskMultiObjects-batchsource.md]: done

File: docs/ZendeskMultiObjects-batchsource.md (hunk 30 above)
Comment (vikasrathee-cs) [reply docs/ZendeskMultiObjects-batchsource.md]: added

File: docs/ZendeskMultiObjects-batchsource.md (hunk 31 above)
Comment (vikasrathee-cs) [reply docs/ZendeskMultiObjects-batchsource.md]: done

File: docs/ZendeskMultiObjects-batchsource.md (hunk 32 above)
Comment (vikasrathee-cs) [reply docs/ZendeskMultiObjects-batchsource.md]: done

File: docs/ZendeskMultiObjects-batchsource.md (hunk 33 above)
Comment (vikasrathee-cs) [reply docs/ZendeskMultiObjects-batchsource.md:101]: only single space is there

**Comment by vikasrathee-cs** [comment]
Closing this PR and raising a separate PR as it is showing merge commits. https://github.com/data-integrations/zendesk/pull/30

**Pull request closed without merging**